pkgs/applications/misc/dump1090/default.nix

```diff
@@ -0,0 +1,35 @@
+{ stdenv, fetchgit, gnumake, pkgconfig, libusb, rtl-sdr }:
+
+stdenv.mkDerivation rec {
+  name = "dump1090";
+
+  src = fetchgit {
+    url = "https://github.com/MalcolmRobb/dump1090";
+    rev = "bff92c4ad772a0a8d433f788d39dae97e00e4dbe";
+    sha256 = "06aaj9gpz5v4qzvnp8xf18wdfclp0jvn3hflls79ly46gz2dh9hy";
+  };
+
+  nativeBuildInputs = [ pkgconfig ];
+  buildInputs = [ gnumake libusb rtl-sdr ];
+
+  buildPhase = ''
+    PREFIX=$out make
+  '';
+
+  installPhase = ''
+    mkdir -p $out/bin
+    cp -v dump1090 $out/bin/dump1090
+  '';
+
+  fixupPhase = ''
+    mkdir -p $out/share
+    cp -vr public_html $out/share/dump1090
+  '';
+
+  meta = with stdenv.lib; {
+    description = "A simple Mode S decoder for RTLSDR devices";
+    homepage = https://github.com/MalcolmRobb/dump1090;
+    license = licenses.bsd3;
```

Comment:
Looks like `bsd2`:

https://github.com/MalcolmRobb/dump1090/blob/bff92c4ad772a0a8d433f788d39dae97e00e4dbe/dump1090.c#L3-L28

Reply:
Oh my bad at the bottom of the readme it says it's released as bsd3. :woman_shrugging: